src/services/zapper.js

```diff
@@ -61,6 +61,10 @@ type ZapperProtocolBalances = {
   }
 };
 
+type ZapperFiatRates = {
```

Comment:
Add comment that the rates are in USD terms

Reply:
Added above method.